pages/common/import.md

```diff
@@ -0,0 +1,15 @@
+# import
+
+> Capture some or all of an X server screen and save the image to a file. Part of ImageMagick library
+
+- Capture the entire X server screen in the Postscript image format:
+
+`import -window root {{filename.postscript}}`
```

Comment:
For png files, you have specified `output.png`, but here you mention `filename.postscript`. I would suggest to keep everything uniform, and mention `output` for all cases.